sysl2/sysl/seqs.go

```diff
@@ -101,12 +101,18 @@ func DoConstructSequenceDiagrams(
 					}
 				}
 
+				bbsAll := seqs.TransformBlackboxesToUptos(bbs, seqs.BbApplication)
```

Comment:
if len(sdEndpoints) ==0, warn "No call statements to build sequence diagram for endpoint %s", endpoint.name